src/main/ui/app/route/About.jsx

```diff
@@ -6,18 +6,33 @@ export default class About extends Component {
     render() {
         return (
                 <div>
-                    <h3>About</h3>
+                    
+                    <h1>About</h1>
 
-                    <p>
-                        <Link to="/indexingStatus">Indexing Status</Link>
+                    <h3>1)How to add your module?</h3>
+
+                    <p> Adding your module to Add Ons for indexing is as easy as creating a pull request!
+                        Before we do that, we need to ensure that your module has been hosted on one of the sites that Add Ons currently supports indexing from: <br/>
+                        a) Bintray <br/>
+                        b) Modulus <br/>
+                    <br/>
+                        Currently we only support these 2 servers but in the future we sure will add more based on user demand! Currently, we are working on adding support for Github Releases as well!
+                    <br/>
+                        Once you have confirmed the above prerequisite, you may head over to the <a target="_blank"
+                           href="https://github.com/openmrs/openmrs-contrib-addonindex/blob/master/PUBLISHING-AN-ADD-ON.md">Publishing an Add on document</a> on Github which contains a comprehensive list of steps.
+                    
                     </p>
 
-                    <p>
-                        <a target="_blank"
-                           href="https://github.com/djazayeri/openmrs-contrib-addonindex/blob/master/README.md">Read more on
-                                                                                                                GitHub</a>
+                    <h3>2)How to contribute to Add Ons development?</h3>
```

Comment:
I suggest rephrasing this as "About the OpenMRS Add Ons tool"

It can have links to:

* Discussion forum (e.g. the new talk category I mentioned above)
* Contribute to the code (e.g. the contributing link)
* License (as mentioned by Robby here https://talk.openmrs.org/t/replacing-modulus-design-thoughts/9619/44)
   * Note that we need to add a license before we go live. Whoops!